src/nidcpower/examples/nidcpower_advanced_sequence.py

```diff
@@ -33,28 +33,33 @@ def example(resource_name, channels, options, voltage_max, current_max, points_p
 
         with session.initiate():
             session.wait_for_event(nidcpower.Event.SEQUENCE_ENGINE_DONE)
-            measurements = session.fetch_multiple(points_per_output_function * 2, timeout=timeout)
+            channels = resource_name.split(',')
+            measurement_group = [None] * len(channels)
+            for i, channel in enumerate(channels):
+                channel_name = channel.strip()
+                measurement_group[i] = session.channels[channel_name].fetch_multiple(points_per_output_function * 2, timeout=timeout)
 
         session.delete_advanced_sequence(sequence_name='my_sequence')
-        line_format = '{:,<4} {:,.6g} {:,.6g} {:<6}\n'
-        print('{:<4} {:<10} {:,<10} {:<6}'.format('Num', 'Voltage', 'Current', 'In Compliance'))
-        i = 0
-        for measurement in measurements:
-            print(line_format.format(i, measurement.voltage, measurement.current, str(measurement.in_compliance)))
-            i += 1
+        line_format = '{:<15} {:<4} {:<10} {:<10} {:<6}'
+        print(line_format.format('Channel', 'Num', 'Voltage', 'Current', 'In Compliance'))
+        for i, measurements in enumerate(measurement_group):
+            num = 0
+            channel_name = channels[i].strip()
+            for measurement in measurements:
+                print(line_format.format(channel_name, num, measurement.voltage, measurement.current, str(measurement.in_compliance)))
+                num += 1
 
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Output ramping voltage to voltage max, then ramping current to current max.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments SMU')
-    parser.add_argument('-c', '--channels', default='0', help='Channel(s) to use')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2/0, PXI1Slot3/1', help='Resource name of National Instruments SMUs')
```

Comment:
`help` needs to be updated

Reply:
The help of which argument? --resource-name? I did update it, from "Resource name of a National Instruments SMU" to "Resource name of National Instruments SMUs". What other changes would like to see?

Reply:
I was thinking about how users can pass fully-qualified channel names and whether "Resource name" is the right word here. Thinking more and checking the concept page in existing docs, I don't think it's wrong to use "Resource name" here. I also can't think of anything better. You can drop this issue.